handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/filter/SimpleTransactionControlFilter.java

```diff
@@ -34,23 +36,36 @@
 public class SimpleTransactionControlFilter extends TransactionControlFilter {
 
   @Override
-  public List<Server> getFilteredListOfServers(List<Server> servers) {
+  public boolean enabled() {
+    return DynamicPropertyFactory.getInstance()
+        .getBooleanProperty("servicecomb.loadbalance.filter.simple.enabled", true).get();
+  }
+
+  @Override
+  public int getOrder() {
+    return 200;
+  }
+
+  @Override
+  public List<Server> getFilteredListOfServers(List<Server> servers, Invocation invocation) {
     List<Server> filteredServers = new ArrayList<>();
     Map<String, String> filterOptions =
```

Comment:
seems this is about configuration and instances
better to move to DiscoveryFilter?

so that we can cache ability?

Reply:
This filter leaves for compatible, maybe we need provide a new one in future. So I leave it here but not remove it. We can think about your idea in future when needed.